build-system/compile/bundles.config.bento.json

```diff
@@ -3,8 +3,7 @@
     "name": "bento-accordion",
     "version": "1.0",
     "options": {
-      "hasCss": true,
-      "npm": true
```

Comment:
are we not publishing this anymore?

Reply:
`npm` is always set to true in the `build-bento` script.